spec/features/people_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+feature 'Person' do
+
+  scenario 'adds a new person' do
+    expect{ create_person }.to change(Person, :count).by(1)
```

Comment:
looks good.

Please fix the application so only Authenticated users can add person, categories etc (this will brake all the specs, but will also force you to test with signed in users in mind, devise already provide some macros for it)